get_limit_combine.sh

```diff
@@ -17,8 +17,6 @@ mo="mx${mx}my${my}mh${mh}"
 pushd Combine  
   echo "Nominal limit being calculated"
   combine --redefineSignalPOI r --cminDefaultMinimizerStrategy 0 --X-rtd MINIMIZER_freezeDisassociatedParams --X-rtd MINIMIZER_multiMin_hideConstants --X-rtd MINIMIZER_multiMin_maskConstraints --X-rtd MINIMIZER_multiMin_maskChannels=2 -M AsymptoticLimits -m ${mh} -d Datacard_${procTemplate}_${m}_${procTemplate}.root -n _AsymptoticLimit_r_${procTemplate}_${mo} --freezeParameters MH,MX,MY --run=blind --setParameters MX=${mx},MY=${my} > combine_results_${procTemplate}_${mo}.txt
-  # Toy limits
-  #combine --redefineSignalPOI r --cminDefaultMinimizerStrategy 0 -m ${mh} -d Datacard_${procTemplate}_${m}_${procTemplate}.root -n _HybridNew_r_${procTemplate}_${mo} --freezeParameters MH,MX,MY -M HybridNew --LHCmode LHC-limits --expectedFromGrid 0.5 --setParameters MX=${mx},MY=${my} --saveHybridResult --rMin=${l} --rMax=${h} --fork 16 -T 500 > combine_results_${procTemplate}_toys_${mo}.txt
```

Comment:
Why delete? It is left on purpose, so that we can have a template command, in case we need to run toys.

Reply:
put back